tensorboard/webapp/BUILD

```diff
@@ -286,6 +286,7 @@ tf_ng_web_test_suite(
         "//tensorboard/webapp/metrics/views/card_renderer:card_renderer_tests",
         "//tensorboard/webapp/metrics/views/main_view:main_view_tests",
         "//tensorboard/webapp/metrics/views/right_pane:right_pane_test",
+        "//tensorboard/webapp/notification_center/_views:_views_test",
```

Comment:
Prefer to create a test target in `notification_center` folder. We no longer want this kind of structure.

Reply:
Adjusted. Please help check again. Thanks!

Reply:
This is not resolve. Please fix.

Reply:
I should've replaced it with `//tensorboard/webapp/notification_center:notification_center_test`
in the following commit https://github.com/tensorflow/tensorboard/pull/4768/commits/56a3359f0e386458aadfdcff0944a311f4e2c84c